src/azure-cli/azure/cli/command_modules/monitor/_params.py

```diff
@@ -343,3 +343,9 @@ def load_arguments(self, _):
         c.argument('intelligence_pack_name', options_list=['--name', '-n'])
         c.argument('workspace_name', options_list='--workspace-name')
     # endregion
+
+    # region monitor clone
+    with self.argument_context('monitor clone') as c:
+        c.argument('source_resource', help="Resource ID of the source resource.")
+        c.argument('target_resource', help="Resource ID of the target resource.")
```

Comment:
I have another concern here. It looks that you only have two parameters here with resource id as values. What about  the subscription in resource id are from two different tenants?

Reply:
list alert in the sub of source resource